backend_jobs/ingestion_pipeline/unit_tests/test_resolution.py

```diff
@@ -0,0 +1,104 @@
+
+"""
+  Copyright 2020 Google LLC
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+"""
+
+import unittest
+from backend_jobs.ingestion_pipeline.providers import image_provider_flickr
+
+
+class TestFlickrProvider(unittest.TestCase):
+
+    first_image = {
```

Comment:
image_with_url_resolution
(so when we read the test we understand what the variable is about with having to go back to declaration)